src/firefly/java/edu/caltech/ipac/firefly/server/ServerCommandAccess.java

```diff
@@ -74,6 +74,7 @@ private static void initCommand() {
         _cmdMap.put(ServerParams.RESEND_EMAIL,           new SearchServerCommands.ResendEmail());
         _cmdMap.put(ServerParams.CREATE_DOWNLOAD_SCRIPT, new SearchServerCommands.CreateDownloadScript());
         _cmdMap.put(ServerParams.UWS_JOB_INFO,           new SearchServerCommands.UwsJobInfo());
+        _cmdMap.put(ServerParams.GET_CAPABILITIES,       new SearchServerCommands.GetCapabilities());
```

Comment:
This is perfectly reasonable to do.  
However, there's another option in which I suggested Jaladh to use for SHA's getObserver's call.
We already have SearchServicesJson.getJsonData() which uses SearchProcessor to return a JSON string.  The advantage of this is not having to expand the list of commands that is already getting big.
I mentioned it here not specifically for this PR, but also for future cases.